test/integration/acceptance/custom/basic/basic.go

```diff
@@ -68,13 +69,41 @@ func (r *BasicTestRunner) Setup(ctx context.Context, createOptsPublic types.Site
 	err = prv1Cluster.VanClient.RouterCreate(ctx, *siteConfig)
 	assert.Assert(t, err)
 
+	var podStartTimeBefore *v1.Time
+	if testSync == true {
+		// Pick the pod details for config-sync validation
+		pods, _ := kube.GetPods("skupper.io/component=router", prv1Cluster.Namespace, prv1Cluster.VanClient.KubeClient)
+		assert.Assert(t, len(pods) > 0)
+		podStartTimeBefore = pods[0].Status.StartTime
+	}
+
 	var connectorCreateOpts types.ConnectorCreateOptions = types.ConnectorCreateOptions{
 		SkupperNamespace: prv1Cluster.Namespace,
 		Name:             "",
 		Cost:             0,
 	}
 	_, err = prv1Cluster.VanClient.ConnectorCreateFromFile(ctx, secretFile, connectorCreateOpts)
 	assert.Assert(t, err)
+
+	var podStartTimeAfter *v1.Time
+	if testSync == true {
+		assert.Assert(t, base.WaitForSkupperConnectedSites(ctx, prv1Cluster, 1))
+
+		linkStatus, err := prv1Cluster.VanClient.ConnectorInspect(context.Background(), "link1")
+		assert.Assert(t, err)
+		if err != nil {
+			log.Fatalf("[TestSync] error: %v", err)
+			return
+		}
+		assert.Assert(t, linkStatus.Connected == true)
```

Comment:
This is a great way to see if the config-sync is working, because it is creating connectors in the router for the links. 

You can also can see that everything is working checking the log, with these messages:
```
2022/04/25 10:24:23 getting sync event
2022/04/25 10:24:53 sync triggered
2022/04/25 10:24:53 sync succeeded
```
`kubectl logs deploy/skupper-router -c config-sync`

But creating links and checking that they are working is enough.
The config-sync also creates httpConnectors /tcpConnectors, so if you expose a service with any protocol and this service is synchronized and available in all the sites, that proves that the config-sync is working as well.

Reply:
I will cover the connectors + config-sync in another test/PR